### ISSUE TITLE
Can the model be used to extract keyword from a paragraph?

### ISSUE THREAD

**Issue by jennifer1995** [ibatra/BERT-Keyword-Extractor#15]
Can the model be used to extract keyword from a paragraph, that consists of several sentences? Thank you.

**Comment by HarrisonLL** [comment]
I went error when using keyword-extractor.py. I essentially changed the input to a file in the test folder. It didn't work and showed this `  File "keyword-extractor.py", line 33, in keywordextract
    attention_mask=segments_tensors)`. 
I am thinking to change the developer's code later, so that we could use this model to predict the whole passage or a phragraph.

**Comment by mr-noman** [comment]
@LidingLi @jennifer1995 have you able to apply this model on whole paragraph to extract keywords?
 Thank you so much.

**Comment by LidingLi** [comment]
No. Later I switched using a model called BERT-KPE. The model is a supervised keyword extraction model and ranks very high in the Microsoft OpenKP competition. It is able to predict keywords (in n-grams) given some input documents.  Consider using that.

**Comment by HarrisonLL** [comment]
No. I later switched to using BERT-KPE. That model can run keyword extraction on a bunch of documents and ranks pretty high in the Microsoft OpenKP competition. Consider using that.

**Comment by mr-noman** [comment]
Alright and Thank you so much @LidingLi
